### PR TITLE
Workers documentation

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

**Comment by GuGuss** [comment]
@Crell Can you add a Changelog entry also. Workers are now deployed and available for PHP.

**Comment by Crell** [comment]
Changelog added, and fixed a broken URL.

**Comment by OriPekelman** [comment]
This needs to be deployed before Wednesday.

File: src/configuration/app/workers.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+The `mail` instance will be a worker that is not web-accessible.
+
+* It will run the `mail-worker.py` script, and restart it automatically if it ever terminates.
+* It will have an environment variable named `TYPE` with value `worker`.
+* It will have no writeable file mounts at all.
+* It will have access only to the RabbitMQ server, through a different relationship name than on the `web` instance.  It will have no access to MySQL whatsoever.
+* It will have "Small" levels of CPU and RAM allocated to it, always.
+
+This way, the web instance has a large upload space, the queue instance has a small amount of scratch space for temporary files, and the mail instance has no writeable disk space at all as it doesn't need it.  The mail instance also does not need any access to the SQL database so for security reasons it has none.  The workers have known fixed sizes, while web can scale to as large as the plan allows.  Each instance can also check the `TYPE` environment variable to detect how it's running and, if appropriate, vary its behavior accordingly.
```
Comment (ralt) [inline src/configuration/app/workers.md]: Maybe this should say "the mail instance has no persistent writeable disk space"? `/tmp` is still available.

Comment (Crell) [inline src/configuration/app/workers.md]: Fixed in production...

**Pull request merged**